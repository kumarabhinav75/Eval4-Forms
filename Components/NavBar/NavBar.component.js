import React, { Component } from 'react';
import {
  SafeAreaView, Image, TouchableOpacity, Text, View
} from 'react-native';

import styles from './NavBar.style';

class NavBar extends Component {
  render() {
    return (
      <View style={styles.navBar}>
        <Text style={styles.navText}>Awesome Forms</Text>
      </View>
    );
  }
}

export default NavBar;
