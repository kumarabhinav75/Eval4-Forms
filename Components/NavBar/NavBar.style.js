import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  navBar: {
    top:0,
    justifyContent: 'space-between',
    backgroundColor: 'darkblue',
    height: 50,
  },
  navText: {
    color: 'white',
    fontSize:30,
    marginLeft: 20,
  }
});
export default styles;
