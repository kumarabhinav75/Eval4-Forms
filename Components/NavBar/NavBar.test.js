import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from './NavBar.component';

describe(NavBar, () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<NavBar />);
    expect(tree).toMatchSnapshot();
  });
});
