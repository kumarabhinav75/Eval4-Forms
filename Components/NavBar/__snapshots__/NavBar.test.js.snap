// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "darkblue",
      "height": 50,
      "justifyContent": "space-between",
      "top": 0,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 30,
        "marginLeft": 20,
      }
    }
  >
    Awesome Forms
  </Text>
</View>
`;
